lib/block-supports/elements.php

```diff
@@ -65,6 +64,43 @@ function gutenberg_render_elements_support( $block_content, $block ) {
 		$content              = substr_replace( $block_content, ' class="' . $class_name . '"', $first_element_offset + strlen( $first_element ) - 1, 0 );
 	}
 
+	return $content;
+}
+
+/**
+ * Render the elements stylesheet.
+ *
+ * @param string|null   $pre_render   The pre-rendered content. Default null.
+ * @param array         $parsed_block The block being rendered.
+ */
+function gutenberg_render_elements_support_footer( $pre_render, $block ) {
```

Comment:
Can we add `@return null` as PHPDoc and make sure we return `null` explicitly in every place so it's clear that this function doesn't short-circuit the `render_block` filter?

It'd be great to add a paragraph in the description explaining why this is the way it is. A suggestion:

> In the case of nested blocks we want the parent element styles to be rendered before their descendants. This solves the issue of an element (e.g.: link color) being styled in both the parent and a descendant: we want the descendant style to take priority, and this is done by loading it after, in DOM order.

Reply:
Done in 711d8f5